.github/workflows/combine-prs.yml

```diff
@@ -12,49 +12,70 @@ on:
     inputs:
       branchPrefix:
         description: 'Branch prefix to find combinable PRs based on'
-        required: true
+        required: false
+        type: string
         default: 'dependabot'
       mustBeGreen:
         description: 'Only combine PRs that are green (status is success)'
-        required: true
+        required: false
+        type: boolean
         default: true
       combineBranchName:
         description: 'Name of the branch to combine PRs into'
-        required: true
+        required: false
+        type: string
         default: 'combine-prs-branch'
       ignoreLabel:
         description: 'Exclude PRs with this label'
-        required: true
+        required: false
+        type: string
         default: 'nocombine'
+      runsOn:
+        description: 'Machine to run the workflow on.'
+        required: false
+        type: string
+        default: '["self-hosted", "fear-ephemeral"]'
```

Comment:
Since [Github recommends to not use self-hosted runners in public repositories](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#self-hosted-runner-security), I would suggest to not make the runners part of the inputs, and specially not having the default value as self-hosted.

Reply:
That makes sense. However the runner needs to be part of the input, otherwise you can not change it (you can not specify `runs-on` in the action that uses a reusable workflow, only if it uses an action). I will not default to `ubuntu-latest` not to run on Github-provided runners unintentionally.

I will remove the default value:
```suggestion
```

Then each action calling this reusable workflow will provide its own runners:
```
    uses: Typeform/.github/.github/workflows/combine-prs.yml@combine-prs
    with:
      runsOn: '["self-hosted", "fear-ephemeral"]'
```

Does this make sense?

Reply:
> you can not specify runs-on in the action that uses a reusable workflow, only if it uses an action

Oh, I didn't know about this.

LGTM!